core/templates/dev/head/pages/base.html

```diff
@@ -249,6 +255,9 @@
     <script src="{{TEMPLATE_DIR_PREFIX}}/components/social_buttons/SocialButtonsDirective.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/components/top_navigation_bar/TopNavigationBarDirective.js"></script>
 
+    <script src="{{TEMPLATE_DIR_PREFIX}}/components/summary_tile/CircularImageDirective.js"></script>
```

Comment:
I missed this, sorry. We are trying not to put imports which are not needed in base.html anymore, since they end up getting loaded for all pages, often unnecessarily. Can you put them only in pages where they are needed? /cc @MAKOSCAFEE